src/core/Object3D.d.ts

```diff
@@ -293,6 +293,11 @@ export class Object3D extends EventDispatcher {
 	 */
 	attach( object: Object3D ): this;
 
+	/**
+	 * Removes object as a child of this, while maintaining the object's world transform.
+	 */
+	detach( object: Object3D ): this;
+
```

Comment:
It seems your branch is based on an old commit. The TS type declaration files were removed quite a while ago.

I suggest you rebase your branch and force-push to fix the PR.

Reply:
I will. Strange, I would swear I just forked this.